src/views/CheckoutSuccess.vue

```diff
@@ -1,27 +1,30 @@
 <template>
     <div class="success center">
         <div class="icon-checkmark-circle"></div>
-        <h1>Your {{ requestKindString }}<br>was successfull!</h1>
+        <h1>Your payment<br>was successfull!</h1>
```

Comment:
Since both types of transactions now have a separate success component, the dynamic type is here not needed anymore.